backend/backend/api/routes/room.py

```diff
@@ -0,0 +1,93 @@
+from functools import lru_cache
+from typing import Annotated, Union
+
+from fastapi import APIRouter, Depends, status, Response
+
+
+from backend.common.room import Room
+from backend.common.player import Player
+from backend.api.models.room import RoomId
+from backend.api.models.player import PlayerId
+from backend.api.routes.player import get_players
+
+router = APIRouter()
+
+@lru_cache
+def get_rooms() -> dict:
+    return {}
+
+
+@router.post("/create_room/", tags=["room"], status_code=status.HTTP_201_CREATED,
+             response_model=RoomId)
+async def create_room(player_id: PlayerId,
+                      rooms: Annotated[dict[str, Room], Depends(get_rooms)],
+                      players: Annotated[dict[str, str], Depends(get_players)],
+                      response: Response):
+    room = Room()
+    player_name = players.get(player_id.player_id)
+
+    if not player_name:
+        response.status_code = status.HTTP_404_NOT_FOUND
+        return f"no user under given id {player_id.player_id}"
```

Comment:
Always return JSON, even if there is an error, JSON are easier to extend and are more efficient in web communications where there is a certain need to understand what was the issue we might encounter

Reply:
done